insights/parsers/ansible_tower_config_custom.py

```diff
@@ -0,0 +1,44 @@
+"""
+AnsibleTowerConfigCustom - file ``/etc/tower/conf.d/custom.py``
+===============================================================
+The AnsibleTowerConfigCustom class parses the file ``/etc/tower/conf.d/custom.py``.
+"""
+from insights import parser, get_active_lines, Parser, LegacyItemAccess
+from insights.core.filters import add_filter
+from insights.specs import Specs
+from insights.parsers import SkipException
+
+add_filter(Specs.ansible_tower_config_custom, ["AWX_CLEANUP_PATHS", "="])
```

Comment:
@wushiqinlou filters should only be added to a parser if they are necessary for the parser to work correctly.  For instance if you are parsing a INI-type config file with headings like `[section_name]` then you would need to add a filter to the parser `[` to ensure that the config file parser won't raise an exception.  In this case rules using this parser should be adding any necessary filters.  

Also, `=` is not a good filter since it will collect too many lines from the file.

Reply:
@bfahr - In the former code, I checked all of the content without filter. However, per discussion with @xiangce , the file "/etc/tower/conf.d/custom.py" is python file, it is possible that the content format will change, then we will get unexpected result. As the new rule only needs to check option "AWX_CLEANUP_PATHS" now and sensitive information may be configured in such file, we decide to add filter into this parser.

Reply:
I agree that the spec needs to be filtered, but my recommendation is that the filter should only be added by rules using the parser and not directly by the parser.  Parsers should be general purpose and by adding that filter you are making it specific to a particular rule.  The only parsers that should be adding filters for a filtered spec are those that require certain information to function properly as I described above in the INI file parser example.

Reply:
@bfahr - ACK, removed the filter from the parser.